solr/contrib/ltr/example/train_and_upload_demo_model.py

```diff
@@ -10,22 +10,24 @@
 
 solrQueryUrl = ""
 
-def generateQueries(config):
-        with open(config["userQueriesFile"]) as input:
+def generateQueries(userQueriesFile,collection,requestHandler,solrFeatureStoreName,efiParams):
+        with open(userQueriesFile) as input:
```

Comment:
minor: `open(userQueriesFile, 'r')` for being consistent with `libsvm_formatter.py` style?